cmd/influx/replication.go

```diff
@@ -66,9 +66,13 @@ func newReplicationCreateCmd() cli.Command {
 			&cli.StringFlag{
 				Name:        "remote-bucket-id",
 				Usage:       "ID of remote bucket data should be replicated to",
-				Required:    true,
 				Destination: &params.RemoteBucketID,
 			},
+			&cli.StringFlag{
+				Name:        "remote-bucket-name",
```

Comment:
In all other commands that take a bucket name or bucket ID, the flags are `--bucket` and `--bucket-id`. Should this flag just be `--remote-bucket`?